CONTRIBUTING.adoc

```diff
@@ -0,0 +1,184 @@
+
+=== Working group meetings
+
+The https://github.com/knative/community/blob/master/working-groups/WORKING-GROUPS.md#documentation[Knative Documentation Working Group] meets biweekly on Tuesdays at 9:30am PT. https://calendar.google.com/calendar/embed?src=knative.team_9q83bg07qs5b9rrslp5jor4l6s%40group.calendar.google.com[Click here] to see the exact dates on the Knative working group calendar.
+
+The Working Group meetings are used to discuss documentation strategy and policy; we expect individual PR review to happen mostly asynchronously via GitHub, and https://slack.knative.dev[Slack].
+
+=== Team interface
+
+Documentation is a collaboration between several different disciplines, including engineers writing code, technical writers providing clarity and structure, product managers and marketers providing vision and comparisons, and readers who may contribute ideas and requests for clarification or additional edits.
+
+With many different types of contributors, it’s useful to establish a ``team API'' or process for the contributors interacting. Note that this ``API contract'' is a living document, and records the current best practices for operating efficiently across timezones without requiring too much coordination.
+
+Generally, we expect the tech writing team to own much of the process and results of the documentation process, harnessing the energy of other contributors and using tech writers as expert resources to improve the quality of existing documentation rather than writing content from scratch.
+
+== Workflow
+
+There are roughly two workflows: writing / contributing new content, and refining and reviewing existing content.
+
+=== New Content
+
+We expect most new content to be written by the subject matter expert (SME) which would be the contributor who actually worked on the feature or example.
+
+When writing new content, keep the following in mind:
+
+* Write one <<type of documentation>> per page.
+* If possible, follow the style/template of other documents of the same type.
+* Focus mostly on technical correctness; structure and language should be roughly correct, but don't need heavy review.
+
+The goal of adding new content is to get technically correct documentation into the repo before it is lost. Tech Writers may provide some quick guidance on getting documentation into the correct location, but won't be providing a detailed list of items to change.
+
+=== Refining and Reviewing Content
+
+Once the raw documentation has made it into the repo, tech writers and other communications experts will review and revise the documentation to make it easier to consume. This will be done as a second PR; it's often easier for the tech writers to clean up or rewrite a section of prose than to teach an engineer what to do, and most engineers would prefer to get back to working on code.
```

Comment:
```suggestion
Once the raw documentation has made it into the repo, tech writers and other communications experts will review and revise the documentation to make it easier to consume. This will be done as a second PR; it's often easier for the tech writers to clean up or rewrite a section of prose than to teach an engineer what to do, and most engineers trust the wordsmithing the tech writers produce more than their own.
```

Reply:
I like what Scott is going for here. This came up in Docs WG today and it seems to me it's best to leave the wordsmithing to the wordsmithers :D

Reply:
Maybe something like "rewrite a section of prose than to teach an engineer what to do, and if they don't have any clarifying questions they will commit changes for style and clarity"

Reply:
Thanks, I like Scott's version much better.